Adds documentation for new integration flexit_bacnet (#29939)
home-assistant/home-assistant.github.io: source/_integrations/flexit_bacnet.markdown

---
title: Flexit BACnet
description: Instructions on how to integrate Flexit air handling unit into Home Assistant.
ha_category:
  - Climate

> ha_release: 2023.12

Reviewer: ```suggestion
ha_release: 2024.1
```

ha_iot_class: Local Polling
ha_domain: flexit_bacnet
ha_platforms:
  - climate
ha_integration_type: integration
---

Integrates [Flexit](https://www.flexit.no/en/) Nordic series air handling unit into Home Assistant.

## Prerequisites

> To configure the integration, you need to obtain IP address and Device ID for the unit.

Reviewer 2: ```suggestion
To configure the integration, you need to obtain the IP address and Device ID for the unit.
```

> 1. Open Flexit Go app on your mobile.

Reviewer 2: ```suggestion
1. Open the Flexit Go app on your mobile.
```

> 2. Use "Find product" button on tha main screen.

Reviewer 2: ```suggestion
2. On the main screen, select the **Find product** button.
```

> 3. Select your device and press "Connect".

Reviewer 2: ```suggestion
3. Select your device and select **Connect**.
```

> 4. Enter installer code (default: 1000) and press "Login".

Reviewer 2: ```suggestion
4. Enter the installer code (default: 1000) and select **Login**.
```

> 5. Open "More" menu -> Installer -> Communication -> BACnet settings.

Reviewer 2: ```suggestion
5. Go to **More** > **Installer** > **Communication**  > **BACnet settings**.
```

> 6. Note down "IP address" and "Device ID".

Reviewer 2: ```suggestion
6. Note down the **IP address** and **Device ID**.
```
